### ISSUE TITLE
deauth is not functional

### ISSUE THREAD

**Issue by ghost** [sankethj/ZiFi#1]
does not deauth ap's

**Comment by sankethj** [comment]
Are you trying deauth on mobile hotspot or on router?

**Comment by EPOXUHA** [comment]
It is clear you copied it from M1z23R source, it has the same issue

**Comment by sankethj** [comment]
> It is clear you copied it from M1z23R source, it has the same issue

You are supposed to speak about the issue here not taunting me of copying thing. 
And check properly in video and readme file, I have mentioned that this is his modified script. I would have made pull req if he had license in his repo.  
And I'm working on this to add some more features after my exams, so just don't simply say clearly copying without going through the code, it really kinda hurts man💔.

**Comment by nyihan** [comment]
please solve this bug. i lile your evil-twin interface

**Comment by k3tiv** [comment]
indeed, on NodeMCU ESP8266 deauth function doesnt works, please fix it
deauth works in project https://github.com/SpacehuhnTech/esp8266_deauther project, but there is no 'evil-twin' function :(

**Comment by Searchanik** [comment]
Deauth not working.
